docs/package/gluon-config-mode-geo-location.rst

```diff
@@ -12,3 +12,11 @@ config_mode.geo_location.show_altitude : optional
     - ``true`` enables the altitude field
     - ``false`` disables the altitude field if altitude has not yet been set
     - defaults to ``true``
+
+Example::
+
+  config_mode = {
+    geo_location = {
+      show_altitude = true
```

Comment:
we should be consistent in whether we add a comma after the last item or not.
i suggest you add one here too, like it is in most places in the docs